_includes/social/icon_partial.html

```diff
@@ -1,7 +1,8 @@
 {% if include.isDisplayed %}
 <li>
     <a href="{{ include.url }}"
-       title="{{ site.data.language.str_follow_on }} {{ include.social }}">
+       title="{{ site.data.language.str_follow_on }} {{ include.social }}"
+       rel="me">
```

Comment:
what's that `rel="me"` attribute?

Reply:
Ok saw the link

Reply:
Mastodon read. It is used to verify authenticity of link.